core/templates/pages/exploration-player-page/exploration-player-page.component.ts

```diff
@@ -58,6 +59,41 @@ angular.module('oppia').component('explorationPlayerPage', {
             angular.element('meta[property="og:description"]').attr(
               'content', response.exploration.objective);
           });
+
+        var bindExplorationPlayerShortcuts = function() {
+          Mousetrap.bind('s', function() {
+            document.getElementById('skipToMainContentId').focus();
+            return false;
+          });
+
+          Mousetrap.bind('k', function() {
+            var previousButton = document.getElementById('backButtonId');
+            if (previousButton !== null) {
+              previousButton.focus();
+            }
+            return false;
+          });
+
+          Mousetrap.bind('j', function() {
+            var nextButton = <HTMLElement>document.querySelector(
+              '.protractor-test-next-button');
+            var continueButton1 = <HTMLElement>document.querySelector(
```

Comment:
Please give it a better and more descriptive name. eg. contineToNextCardButton
Ditto below.